CMakeLists.txt

```diff
@@ -31,4 +38,32 @@ target_link_libraries(ftxui-starter
   PRIVATE ftxui::component # Not needed for this example.
 )
 
-install(TARGETS ftxui-starter RUNTIME DESTINATION "bin")
+add_custom_command(TARGET ${PROJECT_NAME} 
```

Comment:
I think you only have to copy the "index.html" and the python script into the build directory.

Could you use:
https://github.com/ArthurSonzogni/termBreaker/blob/6c882381ca41f2cce693e3cb39284d6c281ed5b4/CMakeLists.txt#L216-L220

instead of all those lines? I worry they are going to afraid newcomers ;-)

Reply:
I understand your worries, I personally feel that a newcomer would enjoy to have a clear list of files he need to keep and files he can dump (build files) having a target file (or any other name) makes it easier to identify mandatory files for its project to work (at least for wasm part)

waiting for your answer to remove or not this part